src/styles/main.scss

```diff
@@ -1,3 +1,212 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 300;
+  font-size: 12px;
+}
+
+.header {
```

Comment:
checklist 8th item, create different files for different blocks, add them for all the blocks